### DISCUSSION TITLE
Rewrite from app 1 to app 2 can't load layout, page when app 2 using next-intl

### DISCUSSION THREAD

**Discussion by whistledev0910** [amannn/next-intl#352] [Q&A]
### Description

Hello,
Thank you for this great package!

When I don't use multi-language, my application works perfectly fine. So I'm speculating that I'm having a problem with next-intl. App 1 cannot load resources from app 2 via rewrite in next.config.js. Resource pages are saying cannot be downloaded from _next/static/css/app/[locale]/**/*.[css,js].

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://github.com/vercel/examples/tree/main/solutions/microfrontends

### Reproduction description

1. Using tuborepo [template](https://github.com/vercel/examples/tree/main/solutions/microfrontends)
2. App 2 i'm using next-inlt
3. App 1 rewrite /any-thing to [app2-url]/en/home

Video demo [here](https://veed.io/view/a932ac50-dfb0-4ffc-8920-db9d5e96aa9f).

### Expected behaviour

Using rewrite to the application that uses next-intl

**Comment by amannn** [comment]
I'll have to convert this to a discussion as it appears to be a usage question and not a bug report, I hope you understand!

**Comment by armspkt** [comment]
@whistledev0910 @amannn 
Is there any solution to fix this?

I had the same issue with creating two next.js applications and want to rewrite some path of app1 to app2

such as
localhost:3000/en/test -> app1
localhost:3001/test2 -> app2

when I go to 
localhost:3000/en/test2 from app1
I want to rewrite everything to localhost:3001/test2.

thanks.
